api/rosetta/data_integration_test.go

```diff
@@ -54,8 +56,14 @@ func setupDB(t *testing.T) *badger.DB {
 	require.NoError(t, err)
 
 	reader := hex.NewDecoder(strings.NewReader(snapshots.Rosetta))
+	dict, _ := hex.DecodeString(dictionaries.Payload)
 
-	err = db.Load(reader, runtime.GOMAXPROCS(0))
+	decompressor, err := zstd.NewReader(reader,
+		zstd.WithDecoderDicts(dict),
+	)
+	require.NoError(t, err)
+
+	err = db.Load(decompressor, runtime.GOMAXPROCS(0))
```

Comment:
Isn't it clearer to use [runtime.NumCPU](https://golang.org/pkg/runtime/#NumCPU) instead here? AFAIK it does the same thing but it's a bit clearer, since `runtime.GOMAXPROCS` can also be used to actually change the amount of max procs